examples/prompt-tuning-personachat.ipynb

```diff
@@ -0,0 +1,312 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a07e0f5e",
+   "metadata": {},
+   "source": [
+    "<div>\n",
+    "<img src=\"https://camo.githubusercontent.com/473dd9f992924d27457650251786464f72e54121ac6e9210add0f483ca849277/68747470733a2f2f692e696d6775722e636f6d2f3765523750616e2e706e67\" width=\"40%\">  \n",
+    "</div>\n",
+    "\n",
+    "# Distributed Bloom for Text Generation using Prompt Tuning\n",
+    "\n",
+    "In this example, we showcase how the a test 6B version of BLOOM model can be efficiently adapted in a decentralized fashion using Petals. In particular, servers maintain the Bloom transformer, which is kept unchanged during adaptation, and learn only a few prefix tokens.\n",
+    "\n",
+    "This example will train the BLOOM model for the chatbot task. In a given dialogue context the model has to provide a relevant answer. [Link to dataset](https://huggingface.co/datasets/bavard/personachat_truecased)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a3f8526f",
+   "metadata": {},
+   "source": [
+    "First, we have to prepare all dependencies."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b4ab6ca7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%env OMP_NUM_THREADS=24\n",
+    "\n",
+    "import os\n",
+    "import sys\n",
+    "sys.path.insert(0, \"..\")\n",
+    "\n",
+    "# General \n",
+    "import torch\n",
+    "from tqdm import tqdm\n",
+    "from torch.optim import AdamW\n",
+    "from torch.utils.data import DataLoader\n",
+    "\n",
+    "# Distributed\n",
+    "from src.client.remote_model import DistributedBloomForCausalLM\n",
+    "\n",
+    "# HF imports\n",
+    "import transformers\n",
+    "import wandb\n",
+    "from datasets import load_dataset\n",
+    "from transformers import get_scheduler"
```

Comment:
Typically, people divide Python imports into 3 groups:

- Built-in imports
- Third-party imports
- Imports related to this project

I'd suggest to keep it this way and in this order.

```suggestion
    "import torch\n",
    "import transformers\n",
    "import wandb\n",
    "from datasets import load_dataset\n",
    "from tqdm import tqdm\n",
    "from torch.optim import AdamW\n",
    "from torch.utils.data import DataLoader\n",
    "from transformers import get_scheduler",
    "\n",
    "# Import a Petals model",
    "from src.client.remote_model import DistributedBloomForCausalLM\n"
```